src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1PremisesService.kt

```diff
@@ -19,23 +21,63 @@ class Cas1PremisesService(
   val outOfServiceBedService: Cas1OutOfServiceBedService,
   val spacePlanningService: SpacePlanningService,
 ) {
+  companion object {
+    private const val OVERBOOKING_RANGE_DURATION = 12L
+  }
   fun getPremisesSummary(premisesId: UUID): CasResult<Cas1PremisesSummaryInfo> {
     val premise = premisesRepository.findByIdOrNull(premisesId)
       ?: return CasResult.NotFound("premises", premisesId.toString())
 
     val bedCount = premisesService.getBedCount(premise)
     val outOfServiceBedsCount = outOfServiceBedService.getCurrentOutOfServiceBedsCountForPremisesId(premisesId)
+    val overbookingSummary = premise.takeIf { it.supportsSpaceBookings }?.let { buildOverBookingSummary(it) } ?: emptyList()
 
     return CasResult.Success(
       Cas1PremisesSummaryInfo(
         entity = premise,
         bedCount = bedCount,
         availableBeds = bedCount - outOfServiceBedsCount,
         outOfServiceBeds = outOfServiceBedsCount,
+        overbookingSummary = overbookingSummary,
       ),
     )
   }
 
+  private fun buildOverBookingSummary(premises: ApprovedPremisesEntity): List<Cas1OverbookingRange> {
+    val premisesCapacitySummary = spacePlanningService.capacity(
+      premises,
+      range = DateRange(LocalDate.now(), LocalDate.now().plusWeeks(OVERBOOKING_RANGE_DURATION)),
```

Comment:
Very minor proposal - could this constant please be renamed to `OVERBOOKING_RANGE_DURATION_WEEKS`?